package.json

```diff
@@ -35,55 +35,54 @@
   ],
   "dependencies": {
     "@salesforce/bunyan": "^2.0.0",
-    "@salesforce/kit": "^1.5.8",
-    "@salesforce/schemas": "^1.0.1",
+    "@salesforce/kit": "^1.5.28",
+    "@salesforce/schemas": "^1.1.0",
     "@salesforce/ts-types": "^1.5.20",
     "@types/graceful-fs": "^4.1.5",
-    "@types/jsforce": "^1.9.29",
-    "@types/mkdirp": "^1.0.1",
+    "@types/mkdirp": "^1.0.2",
     "@types/semver": "^7.3.9",
     "change-case": "^4.1.2",
-    "debug": "^3.1.0",
+    "debug": "^3.2.7",
     "faye": "^1.4.0",
     "form-data": "^4.0.0",
-    "graceful-fs": "^4.2.4",
+    "graceful-fs": "^4.2.9",
     "jsen": "0.6.6",
     "jsforce": "2.0.0-beta.7",
-    "jsonwebtoken": "8.5.0",
+    "jsonwebtoken": "8.5.1",
     "mkdirp": "1.0.4",
     "ts-retry-promise": "^0.6.0"
   },
   "devDependencies": {
-    "@salesforce/dev-config": "^2.1.2",
-    "@salesforce/dev-scripts": "^1.0.2",
+    "@salesforce/dev-config": "^3.0.0",
+    "@salesforce/dev-scripts": "^2.0.0",
     "@salesforce/prettier-config": "^0.0.2",
-    "@salesforce/ts-sinon": "^1.3.15",
-    "@types/debug": "0.0.30",
-    "@types/jsen": "0.0.19",
-    "@types/jsonwebtoken": "8.3.2",
-    "@types/shelljs": "0.7.8",
-    "@typescript-eslint/eslint-plugin": "^4.2.0",
-    "@typescript-eslint/parser": "4.26.0",
-    "chai": "^4.2.0",
-    "commitizen": "^3.0.5",
+    "@salesforce/ts-sinon": "^1.3.21",
+    "@types/debug": "0.0.31",
+    "@types/jsen": "0.0.21",
+    "@types/jsonwebtoken": "8.5.7",
+    "@types/shelljs": "0.8.11",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "4.33.0",
+    "chai": "^4.3.4",
+    "commitizen": "^3.1.2",
     "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^6.15.0",
     "eslint-config-salesforce": "^0.1.6",
     "eslint-config-salesforce-license": "^0.1.6",
-    "eslint-config-salesforce-typescript": "^0.2.7",
-    "eslint-plugin-header": "^3.0.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jsdoc": "^27.0.3",
+    "eslint-config-salesforce-typescript": "^0.2.8",
+    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsdoc": "^27.1.2",
     "eslint-plugin-prettier": "^3.1.3",
     "husky": "^7.0.4",
-    "mocha": "^8.4.0",
+    "mocha": "^9.1.3",
     "nyc": "^15.1.0",
-    "prettier": "^2.0.5",
-    "pretty-quick": "^3.1.0",
-    "shelljs": "0.8.1",
+    "prettier": "^2.5.1",
+    "pretty-quick": "^3.1.3",
+    "shelljs": "0.8.5",
     "sinon": "10.0.0",
-    "ts-node": "^10.0.0",
-    "typescript": "^4.1.3"
+    "ts-node": "^10.4.0",
+    "typescript": "4.4.4"
```

Comment:
I don't think pinning `typescript` to `4.4.4` is the right move here. It leaves us open to issues and/or security vulnerabilities since latest is `4.5.4` right now.

Reply:
I understand Rob, but without the pin we encounter a compile error for which there is no fix. I don't know of an alternative and I am open for suggestions.

Reply:
Approving this now with a work item to repro and create an issue on TypeScript - W-10437470